### DISCUSSION TITLE
Loading in a browser (no module system)

### DISCUSSION THREAD

**Discussion by paulhasell** [microsoft/fluentui#19644] [Q&A] [answered]
Hi, I'm struggling to get v8 working in a browser from unpkg without using a module system. v7 defined a global variable that worked easily but v8 appears not to work that may. This is a breaking change that's not listed. Any advice would be gratefully received.

**Comment by micahgodbolt** [comment, marked as answer]
We use unpkg for our v8 codepen demos, it should work just like v7

https://codepen.io/pen/?template=jOVZXaW
